extensions/stripe/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "stripe",
+  "title": "Stripe",
+  "description": "This extension allows you to make an invoice, create a payment, add a new product, and start a new subscription.",
+  "icon": "stripe.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-a-new-invoice",
+      "title": "Create a new invoice",
```

Comment:
All command titles should use title case ("Create a New Invoice" or just "Create New Invoice"), please also see the [store guidelines on this](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming)

Reply:
Drop the articles, they are redundant in titles like this